### PR TITLE
Router

### PR DESCRIPTION
**Pull request by rodrigoWolox**
## Summary
Routing log in to game and log out implemented

## Screenshot
router

## Trello Card
https://trello.com/c/ozemhzJI/9-routing-i-login

### CODE REVIEW

File: rh-tictactoe/src/app/screens/Game/index.js [hunk 1]
```diff
@@ -20,6 +20,10 @@ class GameContainer extends React.Component {
     this.props.dispatch(actionCreator.jumpTo(value, step));
   };
 
+  logOut = () => {
+    localStorage.removeItem('token');
```
Comment (guidoprinc) [inline rh-tictactoe/src/app/screens/Game/index.js]: this should be in a service, and you should delete any other data too,, like the email and the `Authorization` api header

Comment (rodrigoWolox) [reply rh-tictactoe/src/app/screens/Game/index.js]: Done, logOut now calls an action and cleans all user data. Take into acount that the logout action will be moved to a topbar redux folder in the next branch.

File: rh-tictactoe/src/redux/game/reducer.js [hunk 2]
```diff
@@ -15,9 +15,11 @@ export default function reducer(state = initialState, action) {
       return { ...state, ...action.data };
     case actions.JUMP_TO:
       return { ...state, ...action.data };
+    case actions.LOG_OUT:
+      return { ...state };
```
Comment (guidoprinc) [inline rh-tictactoe/src/redux/game/reducer.js]: you can just don't add it and let default do its work

Comment (guidoprinc) [reply rh-tictactoe/src/redux/game/reducer.js]: { ...state } === state

File: rh-tictactoe/src/services/userService.js [hunk 3]
```diff
@@ -10,4 +10,11 @@ const setInfo = (email, token) => {
   api.setHeader('Authenticated', token);
 };
 
-export { UserService, setInfo };
+const cleanInfo = () => {
+  localStorage.clear();
```
Comment (guidoprinc) [inline rh-tictactoe/src/services/userService.js]: mmmm you can be deleting info you dont want to delete from another application. Beware of this

**Pull request merged**